content/pages/kontakt.md

```diff
@@ -9,7 +9,7 @@ status: published
 Kan vi hjälpa dig att hjälpa klimatet gör vi gärna det. Vi har mångårig
 erfarenhet inom ingenjörskonst och en passion för klimatarbete.
 Eller kanske du vill bli en av oss och bidra och hjälpa till i klimatarbetet,
-kontakta oss då på sociala medier eller skicka ett mail till [info@ingenjorsarbeteforklimatet.se](mailto:info@ingenjorsarbeteforklimatet.se)
+kontakta oss då på sociala medier eller skicka ett mail till [anders.nord@ingenjorsarbeteforklimatet.se](mailto:anders.nord@ingenjorsarbeteforklimatet.se)
```

Comment:
I think info@ingenjorsarbeteforklimatet.se looks better here. Couldn't we forward the emails from that mail account to us three, @strobba @mgcth and @docNord and maintain the same address?